src/components/Modals/TreasurySpendingMotionModal.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+
+import { useDispatch } from 'react-redux';
+import ModalRoot from './ModalRoot';
+import Button from '../Button/Button';
+import { TextInput } from '../InputComponents';
+import { congressActions } from '../../redux/actions';
+import { valueToBN } from '../../utils/walletHelpers';
+import { isValidSubstrateAddress } from '../../utils/bridge';
+
+import styles from './styles.module.scss';
+
+function TreasurySpendingMotionModal({ closeModal }) {
+  const dispatch = useDispatch();
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm({
+    mode: 'all',
+    defaultValues: {
+      spending: 'congressSendLlm',
+    },
+  });
+
+  const onSubmit = ({ transferToAddress, transferAmount }) => {
+    dispatch(
+      congressActions.congressSendTreasuryLld.call({
+        transferToAddress,
+        transferAmount,
+      }),
+    );
+    closeModal();
+  };
+
+  return (
+    <form
+      className={styles.getCitizenshipModal}
+      onSubmit={handleSubmit(onSubmit)}
+    >
+      <h3 className={styles.h3}>Create new spending</h3>
+      <span className={styles.description}>
+        Here you can create a new motion for LLM spending.
+      </span>
+
+      <span className={styles.title}>Recipient Address</span>
+      <TextInput
+        register={register}
+        name="transferToAddress"
+        required
+        errorTitle="Address not valid"
+        validate={(v) => isValidSubstrateAddress(v) || 'Invalid length'}
+      />
+      {errors?.transferToAddress?.message && (
+        <div className={styles.error}>{errors.transferToAddress.message}</div>
+      )}
+      <span className={styles.title}>Amount</span>
+      <TextInput
+        register={register}
+        name="transferAmount"
+        required
+        errorTitle="Amount not valid"
+        validate={(v) => {
+          try {
+            if (valueToBN(v).lt(0)) {
```

Comment:
Why `valueToBN` instead of standard parseFloat?
Also I see that we've missed the same thing in previous reviews. Can you fix this in `SpendingMotionModal.js` as well? Can be in this PR.